test_integration/skipped_all_assertion_test.exs

```diff
@@ -0,0 +1,14 @@
+# exit: 2
+defmodule Mneme.Integration.SkippedAllAssertionTest do
+  use ExUnit.Case
+  use Mneme
+
+  test "skipping all tests do not update code but do cause an error exit at end" do
+    # S
+    auto_assert 2 + 2
+
+    auto_assert 2 + 2
+
+    auto_assert 1 + 2
+  end
```

Comment:
Minor rewording and let's also add one more test that shouldn't run because of the skip-all.

```suggestion
  test "skipping all tests does not update code but does exit with an error code" do
    # S
    auto_assert 2 + 2

    auto_assert 2 + 2

    auto_assert 1 + 2
  end
  
  test "shouldn't run because skip all was used in the first test" do
    auto_assert 1 + 1
  end
```